content/3_Silver/Two_Pointers.mdx

```diff
@@ -280,3 +280,103 @@ public class Books {
 ## Problems
 
 <Problems problems="ad" />
+
+## Quiz
+
+<Quiz>
+  <Quiz.Question>
+    What is the two pointers algorithm?
+    <Quiz.Answer>
+      Keeping track of two indices to find a subarray that satisfies some condition.
+      <Quiz.Explanation>
+        Incorrect. The indices don't necessarily have to represent a subarray, they could just represent elements in the array.
```

Comment:
isn't a single element a subarray (or do you mean subsequence?)

Reply:
shoot, meant subsequence